sass_processor/management/commands/compilescss.py

```diff
@@ -83,6 +83,13 @@ def add_arguments(self, parser):
             help=_(
                 "Set the precision for numeric computations in the SASS processor. Default: settings.SASS_PRECISION.")
         )
+        parser.add_argument(
+            '--exclude-py',
+            action='store_true',
+            dest='exclude_py',
+            default=False,
+            help=_("Exclude python files from scan?")
```

Comment:
Nitpick: perhaps change question mark to dot.

Reply:
Rather, no punctuation mark at all.